app/templates/sites/index.hbs

```diff
@@ -43,9 +44,9 @@
       </td>
       <td>{{site.siteType.label}}</td>
       <td>
-        <AuLink @model={{site.id}} @route="sites.site.edit">
```

Comment:
If we are going to comment out the link I wouldn't remove the route so it works when uncommented